src/main/java/pl/cinema/server/reservation/dto/MakerDto.java

```diff
@@ -0,0 +1,26 @@
+package pl.cinema.server.reservation.dto;
+
+import lombok.Builder;
+import lombok.Value;
+import pl.cinema.server.reservation.validation.PersonName;
+import pl.cinema.server.reservation.validation.Surname;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Value
+@Builder
+public class MakerDto {
+    @NotBlank
+    @PersonName
+    String name;
+
+    @NotBlank
+    @Surname
+    String surname;
+
+    @NotNull
```

Comment:
Skoro już wstawiamy wiek, to może weryfikacja czy jest dodatni?

Reply:
tak, racja. @Min z wartością 0 lub 1 było by super

Reply:
cóż, 0 lat raczej nie, więc minimum 1 rok

Reply:
Czyli z półrocznym dzieckiem byś mnie nie wpuścił na salę?